pybossa/view/projects.py

```diff
@@ -3732,7 +3732,8 @@ def integrate_ext_config(config_dict):
     if request.method == 'POST':
         try:
             data = json.loads(request.data)
-            project.info['ext_config'] = integrate_ext_config(data.get('config'))
```

Comment:
Would a simpler fix be to default to an empty dict using `data.get('config', {})`?

```python
project.info['ext_config'] = integrate_ext_config(data.get('config', {}))
```

Reply:
> Would a simpler fix be to default to an empty dict using `data.get('config', {})`?
> 
> ```python
> project.info['ext_config'] = integrate_ext_config(data.get('config', {}))
> ```

The idea is to skip calling `integrate_ext_config` when input passed doesn't contain config whereas the code described above still make call to `integrate_ext_config` possibly resulting in overwriting empty config.